src/assets/content/dictionary/open-source-software-oss.md

```diff
@@ -1,6 +1,12 @@
 ---
 title: Open Source Software (OSS)
-definition: ''
-perspectives: []
+definition: Open source software is software that is available for anyone to view, copy, change, or distribute for any purpose. Some notable examples of open source software include the Linux operating system, the Mozilla Firefox browser, Apache HTTP Server software, and LibreOffice productivity software. Every year, open source software contributions are celebrated during October as part of a month-long event called Hacktoberfest
+
+sources: 
+- sourceurl: https://en.wikipedia.org/wiki/Open-source_software, https://hacktoberfest.com
+perspectives: 
+- meaning: is an important part of the software development community, allowing collaboration of large numbers of programmers with diverse perspectives, increasing the speed of innovation 
```

Comment:
Hi @laurasimsdev ! Thank you so much for your contribution to our tech dictionary. I have just a couple small request to get your changes ready for approval.

Please take a look at our [CONTRIBUTING.md](https://github.com/cherryontech/website/blob/pit/CONTRIBUTING.md) to see the necessary formatting. The `meaning` field will be a part of an existing sentence structure, so we can see the perspective sentence doesn't read quite right when we test your change (via Netlify):
<img width="1770" alt="image" src="https://user-images.githubusercontent.com/3057984/197912829-9f28fe96-733e-41f2-b881-91367fe93ef8.png">

I strongly suggest following the directions listed in the CONTRIBUTING.md for testing locally or via the Netlify deploy preview in order to read your sentence. This helps you catch mistakes like the additional period and capitalization, as well as reading the full sentence for clarity.